trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative './base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name[0..9] if @nameable.correct_name.length > 10
+  end
```

Comment:
- [ ] Kindly provide and else case since now it will return `nil` if the `name` is less than or equal `10`.